docs/design/adapt-k8s-todo.md

```diff
@@ -0,0 +1,29 @@
+# Things to do when volcano adapts to k8s version upgrade
+
+## Motivation
+
+In order to ensure that volcano is compatible with the new version of kubernetes, the k8s version of volcano needs to be routinely upgraded.
+
+
+## TODO things for upgrade
+
+####  Upgrade volcano apis
+First,we need to upgrade the k8s version of volcano apis, such as:https://github.com/volcano-sh/apis/pull/124.

```

Comment:
Here should list what we need to do when update apis repo.
And also add to add that go version in go mod file and go build version of dockerfile should keep consistent with k8s.

Reply:
done